tasks/ubuntu/bionic.task/store_ip.erb

```diff
@@ -0,0 +1,12 @@
+<%# -*- shell-script -*- %>
+# Get current IP
+node_ip=$(ip addr show | grep -A 2 BROADCAST | grep inet | awk '{ print $2 }' | awk -F'/' '{print $1}')
+echo IP is $node_ip
+
+echo "$node_ip" > /tmp/store_ip.txt
```

Comment:
Why put this on disk?

Reply:
this was for checking the correct IP only, I have to remove this.